src/ui/uploader.js

```diff
@@ -28,13 +28,20 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
         members: {
             currentFile: null
         },
-        // Holds a fileObjectURL for file preview purposes
         model: {
-            fileObjectURL: null,
-            fileDetails: null
+            // uploadState can be one of the following values:
+            // "ready" (the initial state), "uploading", "errorReceived"
+            uploadState: "ready",
+            fileObjectUrl: null, // for the file preview
+            previousFileObjectUrl: null, // for file deletion on change/upload
+            serverUploadUrl: "/stories/",
```

Comment:
I don't think this should be a model value, as it is likely not something that would change, and isn't part of the "state".